enterprise/models.py

```diff
@@ -2271,19 +2278,58 @@ def render_template(self, template_text, kwargs):
         context = Context(kwargs)
         return template.render(context)
 
+    def __repr__(self):
+        """
+        Return uniquely identifying string representation.
+        """
+        return self.__str__()
+
+
+@python_2_unicode_compatible
+class BulkEnrollmentNotificationEmailTemplate(BaseEnrollmentNotificationEmailTemplate):
+    """
+        Why a new template instead of using 'EnrollmentNotificationEmailTemplate'?
+        1: we do not need customer linking, since template will not be per customer
+        2: we still want to do a model to avoid having template in text/html files on git
```

Comment:
1. Make it optional then
2. Why would you need to do this? If you just add a type column to the existing model you can then map that to your `bulk_enroll_mode` flag and just update the EnrollmentNotificationEmailTemplate uses to also pick that type.

Reply:
Ah I did not want to use the existing model since it's customer specific and needs to fill in a customer uuid. this one is going to be a fixed template (for now anyway) for all bulk enrollment.

Also, for this one I decided it's best to not send emails if there is no template, as opposed to current design of defaulting to the text files (since I would then also have to add text based template as a fallback)... :(

Reply:
Having said that.. there is an option: just exploring...

I confirmed that the `EnrollmentNotificationEmailTemplate` model does require a enterprise_customer to be specified when creating...so the only way to re-use all this setup for bulk enroll, is to also go with text/html based templates as a default fallback for bulk enrollment, and add a column (like type='bulk_enrollment') to the existing model. In bulk enroll mode we find the template that matches that type. Presumably only one exists per type (we can also unique key those fields type + enterprise_customer maybe). When a row with type='bulk_enrollment' is found we use that... then we won't need a new model. When Lael and I looked at this first, it seemed to use it was cleaner to use a new model, but this option can also be made to work. Honestly I am equally sold on both.

What's your pref based on what we know of the use cases? This option seems totally doable as an alternative to the current PR approach of only using one row of the table which I agree is odd.